src/main/java/com/google/api/codegen/configgen/LanguageGenerator.java

```diff
@@ -128,8 +129,9 @@ public String getFormattedPackageName(String packageName) {
       // google.logging.v2 => cloud.google.com/go/logging/apiv2
       // Otherwise, fall back to backup
       if (!isApiGoogleCloud(nameComponents)) {
-        nameComponents.add(0, "google.golang.org");
-        return Joiner.on("/").join(nameComponents);
+        List<String> newComponents = Lists.newArrayList(nameComponents);
```

Comment:
I think it would be easier to follow the code if we used a mutable list everywhere instead of immutable in some places and mutable in others. (I don't think mutability is a concern since the list is local to the function). Concretely, I'm suggesting that the original list be created with `Lists.newArrayList(Splitter...);`